onnxscript/__init__.py

```diff
@@ -23,6 +23,24 @@
 except importlib_metadata.PackageNotFoundError:
     __version__ = None  # type: ignore[assignment]
 
+from .onnx_types import (
+    FLOAT,
+    UINT8,
+    INT8,
+    UINT16,
+    INT16,
+    INT32,
+    INT64,
+    STRING,
+    BOOL,
```

Comment:
Sort from small to big types maybe?

Reply:
It is currently in the onnx enum ordering.

Reply:
But if we want a meaningful order, I would suggest sort by major-category (float, int, uint, etc.) first, and sort by precision next. But not sure this is going to make a big difference.

Reply:
no strong opinion, I think this as-is lgtm. I would just add the isort thing so the linter doesn't complain